frontend/src/Utilities/Http.ts

```diff
@@ -0,0 +1,24 @@
+export const http = (
```

Comment:
Kad jau wrappinam, tai būtų visai cool apwrappint pilnai ir padaryt, kad būtų lengva mockint api call'us.
Siūlyčiau aprašyt interfeisiuką savo ir tada exportint šitą default implementaciją, o po to galėsim lengvai sumockint minimalų server state / duomenis
https://jestjs.io/docs/en/manual-mocks

Reply:
As nelabai tiesa sakant sure, ka tu nori kad pakeisciau. Man atrodo, kad dabar jau galima mockinti callus su tokiu wrapperiu :D Gal galetum numest koki snippeta ka tu turejai omeny?

Reply:
šiaip nesvarbu, žėsim kaip lengviau rašyt testus, kai rašysim testus